### ISSUE TITLE
Failed Creating new product / Accessign to new item after created

### ISSUE THREAD

**Issue by GrupImatica** [salesagility/SuiteCRM#2482]
Hi, after i create a product when i go to details screen i get this error:

Mon Oct 24 23:17:09 2016 [2680][1][FATAL] Error running count query for AOS_Products List:  Query Failed:(
            SELECT count(_) c FROM (
                SELECT
                    aos_quotes._,
                    accounts.id AS account_id,
                    accounts.name AS billing_account,

```
                opportunity_id AS opportunity,
                billing_contact_id AS billing_contact,
                '' AS created_by_name,
                '' AS modified_by_name,
                '' AS assigned_user_name
            FROM
                aos_products

            JOIN aos_products_quotes ON aos_products_quotes.product_id = aos_products.id AND aos_products.id = N'5075ac43-e453-d30a-17ef-580e6c5f51c1' AND aos_products_quotes.deleted = 0 AND aos_products.deleted = 0
            JOIN aos_quotes ON aos_quotes.id = aos_products_quotes.parent_id AND aos_quotes.stage = N'Closed Accepted' AND aos_quotes.deleted = 0
            JOIN accounts ON accounts.id = aos_quotes.billing_account_id -- AND accounts.deleted = 0

            GROUP BY accounts.id
        ) AS aos_quotes

    )::: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]La columna 'aos_quotes.id' de la lista de selección no es válida, porque no está contenida en una función de agregado ni en la cláusula GROUP BY.
```

Mon Oct 24 23:17:09 2016 [2680][1][FATAL] Exception in Controller: Error de base de datos. Por favor, consulte SuiteCRM error .log para más detalles.
#### Your Environment

- SuiteCRM Version used: 7.7.6
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Anything
- Environment name and version (e.g. MySQL, PHP 7): MSSQL , PHP 7
- Operating System and version (e.g Ubuntu 16.04): Windows 2012 R2

**Comment by shogunpol** [comment]
@GrupImatica , please provide details from template, replication steps, screenshot. To replicate the issue i need more than describe what is happen after, i need first to make my instance as much similar as possible. The information what missing to me for example are: what currency you use, do you select product category, and so on.
Thanks for your contribution.

**Comment by GrupImatica** [comment]
Hi, i use Euro, in database is -99 . No i've no selected category. i attach some screenshots:

Creating simple product:

![prod1](https://cloud.githubusercontent.com/assets/20840700/19761547/c13407fc-9c36-11e6-96f7-4333decf13f3.PNG)

After Save:
![prod2](https://cloud.githubusercontent.com/assets/20840700/19761562/cca6519e-9c36-11e6-8faa-178b47124352.PNG)

We go to search in products list:
![prod3](https://cloud.githubusercontent.com/assets/20840700/19761574/d51cde38-9c36-11e6-9914-c21cdb099a35.PNG)

I can't enter:
![prod4](https://cloud.githubusercontent.com/assets/20840700/19761585/dce93116-9c36-11e6-83fc-8b18f1e05485.PNG)

**Comment by shogunpol** [comment]
After additional information i been able to replicate an issue in recent version of suiteCRM(7.7.6), however it appear only in Windows/MSSQL environment.
We label this issue as a "Bug"
@GrupImatica , thanks for your contribution.

**Comment by cfiorillo** [comment]
You have the same error in MySQL when in sql_mode you have ONLY_FULL_GROUP_BY (default from release 5.7.5). The same behavior is in MSSQL and other DB (e.g. Oracle).

To disable this option you can execute:

SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));

My opinion: the offending query needs to be rewritten to remove the error; you need to list all the fields in the select statement in the group by clause, using an aggregate operator like MAX, MIN, SUM when needed

**Comment by Mac-Rae** [comment]
This issue has been closed as version 7.8.x of SuiteCRM has reached its end of life and therefore will no longer be receiving updates. We have also been unable to replicate this issue in any newer version of SuiteCRM such as 7.10.x or 7.11.x however if you are able, please leave a comment explaining how and we'll re-open this issue.

For more information about 7.8.x end of life please visit our [blog post](https://suitecrm.com/7-8-x-end-of-life/)
